### PR TITLE
CaaSP: enable minion blackout on minion onboarding

### PR DESCRIPTION
**Pull request by mateiw**
## What does this PR change?

Automatically enable minion blackout (via Salt formula) when a CaaSP minion is onboarded.

Depends on `salt-blackout` formula https://github.com/SUSE/salt-formulas/pull/29

## GUI diff

After:

image

- [x] **DONE**

## Documentation

- https://github.com/uyuni-project/uyuni-docs/pull/38/

- [x] **DONE**

## Test coverage

- Unit tests were added

- [X] **DONE**

## Links

Tracks https://github.com/SUSE/spacewalk/issues/10312

- [X] **DONE**

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql" (Test skipped, there are no changes to test)
- [ ] Re-run test "java_lint_checkstyle" 		 
- [ ] Re-run test "java_pgsql_tests" 		 
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"		 
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"		 
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: java/code/src/com/redhat/rhn/common/security/acl/Access.java [hunk 1]
```diff
@@ -499,4 +500,18 @@ public boolean aclErrataEditable(Object ctx, String[] params) {
         Errata e = ErrataFactory.lookupById(eid);
         return e != null && e.getOrg() != null;
     }
+
+    public boolean aclFormulaValueEquals(Object ctx, String[] params) {
```
Comment (mbologna) [inline java/code/src/com/redhat/rhn/common/security/acl/Access.java]: Is this used somewhere? I cannot find any reference to it.

Comment (mateiw) [reply java/code/src/com/redhat/rhn/common/security/acl/Access.java]: Yes, here https://github.com/uyuni-project/uyuni/pull/1661/files#diff-85595dcbc480319b138af97b9d2e669aR53

**Comment by mbologna** [comment]
@moio : added you as a reviewer as you already know the `MessageQueue` choice we made

File: java/code/src/com/suse/manager/webui/utils/ViewHelper.java [hunk 2]
```diff
@@ -131,4 +137,25 @@ public String renderDate(Date date) {
...
+        }
+        if (!server.asMinionServer().isPresent()) {
+            return false;
+        }
+        List<String> enabledFormulas = FormulaFactory.getFormulasByMinionId(MinionServerFactory.getMinionId(server.getId()));
+        if (!enabledFormulas.contains(formulaName)) {
+            return false;
+        }
+
+        Map<String, Object> systemData = FormulaFactory.
```
[the remark this answers is not in the document]
Comment (mateiw) [reply java/code/src/com/suse/manager/webui/utils/ViewHelper.java]: The keys from JSON file that store the pillar data.

**Pull request merged**